src/handlers/query-user.ts

```diff
@@ -0,0 +1,70 @@
+import { Context } from "../types/context";
+
+async function checkUserAccess(context: Context, username: string) {
+  const { octokit, payload } = context;
+  if (!payload.organization || !payload.comment.user?.name) {
+    throw new Error("Missing Organization / User from payload, cannot check for organization membership.");
+  }
+  const { status } = await octokit.orgs.checkMembershipForUser({
+    username: payload.comment.user.name,
+    org: payload.organization.login,
+  });
+  // @ts-expect-error Somehow typing seems wrong but according to
+  // https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user--status-codes
+  // 204 means the user is part of the Organization
+  if (status !== 204) {
+    await octokit.issues.createComment({
+      body: `\`\`\`User ${username} cannot request another user as it is not member of the organization.\`\`\``,
```

Comment:
We should standardize messaging through the kernel ideally, or at minimum via our SDK

We should adhere to the `ok`, `info`, `error`, `fatal` message classes with the coloring via the diff styling accordingly as I implemented in v1

Reply:
Agreed. Tried to use the package used in V1 but as it is now it is too tightly coupled with the old version to be reusable. Might wanna do some updates.

Reply:
https://github.com/ubiquity/ubiquibot-logger/issues/2